internal/operands/vm-console-proxy/reconcile_test.go

```diff
@@ -289,9 +388,62 @@ var _ = Describe("VM Console Proxy Operand", func() {
 			ExpectResourceNotExists(serviceAccount, request)
 			ExpectResourceNotExists(bundle.ClusterRole, request)
 			ExpectResourceNotExists(bundle.ClusterRoleBinding, request)
+			ExpectResourceNotExists(bundle.RoleBinding, request)
 			ExpectResourceNotExists(configMap, request)
 			ExpectResourceNotExists(service, request)
 			ExpectResourceNotExists(deployment, request)
+			ExpectResourceNotExists(bundle.ApiService, request)
+		})
+
+		It("should delete Route leftover from a previous version on reconcile", func() {
+			route := &routev1.Route{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      routeName,
+					Namespace: namespace,
+					Annotations: map[string]string{
+						libhandler.TypeAnnotation:           "SSP.ssp.kubevirt.io",
+						libhandler.NamespacedNameAnnotation: namespace + "/" + name,
+					},
+					Labels: map[string]string{
+						common.AppKubernetesNameLabel:      operandName,
+						common.AppKubernetesComponentLabel: operandComponent,
+						common.AppKubernetesManagedByLabel: common.AppKubernetesManagedByValue,
+					},
+				},
+				Spec: routev1.RouteSpec{},
+			}
+
+			Expect(request.Client.Create(request.Context, route)).To(Succeed())
+
+			_, err := operand.Reconcile(&request)
+			Expect(err).ToNot(HaveOccurred())
+
+			ExpectResourceNotExists(route, request)
+		})
+
+		It("should delete Route leftover from a previous version on cleanup", func() {
+			route := &routev1.Route{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      routeName,
+					Namespace: namespace,
```

Comment:
Same as above.

Do these test cases add any value compared to the same ones above?

Reply:
Yes, this case tests the `Cleanup()` method, and the above one tests `Reconcile()` method.

Reply:
done